src/neynar-api/v2/openapi-farcaster/models/cast-with-interactions-all-of-channel.ts

```diff
@@ -0,0 +1,38 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * Farcaster API V2
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 2.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+// May contain unused imports in some cases
+// @ts-ignore
+import { ChannelExternalLink } from './channel-external-link';
+// May contain unused imports in some cases
+// @ts-ignore
+import { ChannelOrDehydratedChannel } from './channel-or-dehydrated-channel';
+// May contain unused imports in some cases
+// @ts-ignore
+import { ChannelUserContext } from './channel-user-context';
+// May contain unused imports in some cases
+// @ts-ignore
+import { User } from './user';
+
+/**
+ * 
+ * @export
+ * @interface CastWithInteractionsAllOfChannel
+ */
+export interface CastWithInteractionsAllOfChannel {
```

Comment:
Something is wrong. This type is empty.

Reply:
Question for @Shreyaschorge

Reply:
Just looked at OAS `main` branch history, and I think the `oas` commit currently checked out by this `nodejs-sdk` is in the territory of a lot of conflicting/mistaken changes being made that Manan corrected today. I think you can skip ahead to `oas` commit `0be47151f427fabacdaa0eaf24d200245485a93f` and go from there.

Reply:
https://github.com/neynarxyz/nodejs-sdk/pull/216 has the same issue.

Reply:
Fixed here. https://github.com/neynarxyz/OAS/pull/243/commits/1347cb0a337f2638e20514dd45b30c2604a25bb9

Yet another one of my changes that was incorrectly reverted.

Reply:
I'll get OAS up to date. Making and merging a bunch of PRs now and will include these cast changes in the last one.